elastic/integrations
# Add in technique.name field to the transform. Remove milliseconds from TQL query.

- Bug
- Enhancement

## Proposed commit message

Add in technique.name field to the transform that came from a customer request. 
Remove milliseconds from TQL query as current code drops the last 0 in the milliseconds which causes as issue. 

## Checklist


## Author's Checklist

- [ ] Confirm new field is consistent across files 

## How to test this PR locally

Test by running against TC with correct data including technique tag on an indicator. Confirm in ES that technique name is included. And then confirm in the ES request logs that milliseconds are not included in API request. 

## Related issues

- 

## Screenshots

-
